HomeWork/src/main/java/generics/Apple.java

```diff
@@ -0,0 +1,19 @@
+package generics;
+
+public class Apple extends Fruit {
+
+    private float weightApple = 1.0F;
```

Comment:
Move this field to the class Fruit and pass the weight to the superclass constructor